pcsx2/GS/GSState.cpp

```diff
@@ -1682,7 +1682,7 @@ void GSState::FlushPrim()
 			{
 				// could be an unsupported draw call
 			}
-			catch (const std::bad_alloc& e)
+			catch (const std::bad_alloc& /* e */)
```

Comment:
Any reason to have it at all?  It's not like knowing we would have named the error `e` helps anyone, given the unhelpfulness of the name `e`

Reply:
I'm not really bothered, I just commented it out quick but I can get rid of it all the same.